Update html.tpl

MDE186 opened the pull request:
Try make mobile a bit more usable

MDE186 committed:
Update html.tpl

Try make mobile a bit more usable

spinza commented:
I'll test it and give feedback.

jammin84 commented:
note: for issue #10

spinza commented:
I've checked this out on https://wind-dev.ctwug.za.net/
I get server error.
The error log has this:

```
2016/01/21 22:57:06 [error] 21271#0: *11804 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Smarty error: [in includes/html.tpl line 24]: syntax error: unrecognized tag: +       var mvp = document.getElementById('windViewport');
+       mvp.setAttribute('content','width=1200'); (Smarty_Compiler.class.php, line 446) in /usr/local/lib/smarty-2.6.29/libs/Smarty.class.php on line 1094" while reading response header from upstream, client: 172.18.236.4, server: wind-dev.ctwug.za.net, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "wind-dev.ctwug.za.net"
```

spinza commented:
@MDE186 do you have a dev enviroment set up?  I can help you set it up.
We run smarty-2.6.29.  That looks like that may be related?

spinza commented on templates/basic/includes/html.tpl:
```diff
@@ -19,7 +19,12 @@
 <head>
 	<meta charset="{$lang.charset}"/>
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta id="windViewport" name="viewport" content="width = 740">
```
Some of these + are literally in the file.  So they are breaking code.

spinza commented:
After removing the + I still get this error:

```
2016/01/21 23:09:34 [error] 21269#0: *11990 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Smarty error: [in includes/html.tpl line 24]: syntax error: unrecognized tag 'var' (Smarty_Compiler.class.php, line 590) in /usr/local/lib/smarty-2.6.29/libs/Smarty.class.php on line 1094" while reading response header from upstream, client: 172.18.236.4, server: wind-dev.ctwug.za.net, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "wind-dev.ctwug.za.net"
```

MDE186 committed:
Update html.tpl

Add {literal} tags to make Smarty use the scripting codes as literal functions.
